server_addon/maya/server/settings/publishers.py

```diff
@@ -309,6 +385,143 @@ class ExtractAlembicModel(BaseSettingsModel):
         description="List of attribute prefixes for attributes that will be "
                     "included in the alembic export.",
     )
+    flags: list[str] = SettingsField(
+        enum_resolver=extract_alembic_flags_enum,
+        title="Export Flags",
```

Comment:
Even though I like the description of what each flag means, just looking at the settings I'm clueless what the "Export flags" setting is for. It seems to be the 'default state' for that particular boolean flag (enabled or disabled). When not 'exposed' to the user the value is _always_ at this default state, otherwise it's exposed to the artist and this would define the default state.

Reply:
Sorry, not sure what you are wanting to change here?

Reply:
It was more of a remark - I went to settings, saw the Export flags. And even with tooltips it didn't click for me what "Export flags" was for and what it was supposed to do.

It was unclear to me that it would define the default state for that flag. As such, my remark was: "Are we ok with it being unclear?" and "Do we see any good way to improve this user experience?"

Usually tooltips explain what the attribute does - but now it's explaining the flags themselves and totally skips explaining what "Export flags" is for. Which is valuable information, but due to it being unclear what "Export flags" was intended to do it did leave me in the dark for a bit.

Reply:
> As such, my remark was: "Are we ok with it being unclear?" and "Do we see any good way to improve this user experience?"

Judge... Leading the witness... :)

I think before doing anything about this, we should reach a decision on the multi-enum in the publisher, cause if that changes we might as well change this to check boxes as well.